apps/nextjs-website/src/_contents/translations.ts

```diff
@@ -30,6 +30,7 @@ export const translations = {
     role: 'Ruolo',
     requiredFieldError: 'Questo campo non può essere vuoto',
     emailFieldError: 'Inserisci un indirizzo email valido',
+    yourData: 'I tuoi dati',
```

Comment:
I see that this key is not used beacause has been moved to it.json i thik we can delete this 
```suggestion
```